### PR TITLE
Adding tag prompt for container and widget generators

### PR DESCRIPTION
**Pull request by Goroner**
This solves #95

### CODE REVIEW

**Comment by mistermark** [comment]
Can we have the prompt ask for a single Section Tag first and then ask for a comma separated array for Regular Tags? I'd like to be able to decide on my own Section tag.
And right now I filled in three tags (comma separated) and I get the following result:

````
<tags>
    <tag type="regular">layout</tag>
    <tag type="section">layouts</tag>
    
    <tag type="container"></tag>
    
    <tag type="testing"></tag>
    
    <tag type="generate"></tag>
    
</tags>
````

File: templates/container/template/model.xml [hunk 1]
```diff
@@ -26,6 +26,9 @@
         <tags>
             <tag type="regular">layout</tag>
             <tag type="section">layouts</tag>
+            <% _.each(tags, function(tag){ %>
+            <tag type="<%= tag%>"></tag>
```
Comment (mistermark) [inline templates/container/template/model.xml]: Move the <%= tag %> as the value of the tag and have the type as 'regular' by default.

File: templates/widget/template/model.xml [hunk 2]
```diff
@@ -19,7 +19,9 @@
             </property> -->
         </properties>
         <tags>
-
+            <% _.each(tags, function(tag){ %>
+            <tag type="<%= tag%>"></tag>
```
Comment (mistermark) [inline templates/widget/template/model.xml]: Move the <%= tag %> as the value of the tag and have the type as 'regular' by default.

**Comment by Goroner** [comment]
@mistermark Check now, updated it as requested.

**Pull request merged**